Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -238,13 +238,9 @@ const std::vector<std::unique_ptr<VideoBackendBase>>& VideoBackendBase::GetAvail
     backends.push_back(std::make_unique<DX12::VideoBackend>());
 #endif
 #ifdef __APPLE__
-    backends.push_back(std::make_unique<Metal::VideoBackend>());
+    backends.emplace(backends.begin(), std::make_unique<Metal::VideoBackend>());
 #endif
 #ifdef HAS_VULKAN
-#ifdef __APPLE__
-    // Emplace the Vulkan backend at the beginning so it takes precedence over OpenGL.
-    backends.emplace(backends.begin(), std::make_unique<Vulkan::VideoBackend>());
-#else
```

Comment:
We still want Vulkan to have precedence over OpenGL.

Reply:
So I should emplace Metal after Vulkan, so that Metal has precedence over Vulkan

Reply:
If they both `emplace(backends.begin(), `, then the second one will take priority.